metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -98,7 +98,10 @@ final class Embedded(
 }
 
 object Embedded {
-  def downloadSettings(dependency: Dependency): Settings =
+  def downloadSettings(
+      dependency: Dependency,
+      forceDependencies: List[Dependency] = Nil
```

Comment:
nit: is this parameter necessary? We can use `withForceVersions` at the call-site instead

Reply:
Ok, added it there. Sorry for the many force pushes - somehow I wasn't formatting although I was sure I was.